TicketSwap/solar
# Instruction component

# Description
This introduces the instruction component that is helpful for when you want to give more context to for example an input element.

## Changes

- [x] Add instruction component

## How to test

- Checkout this branch
- `$ yarn storybook`
- See Components/Inputs/Instruction
